src/components/header/header.three.js

```diff
@@ -65,20 +73,35 @@ export function HeaderLeft({ sections, classes, ...props }) {
             );
           })}
         <React.Fragment>
-          <Button
-            color="primary"
-            variant="outlined"
-            size="small"
-            aria-label="Get Listed"
-            component={props => (
-              <Link to="/app/profile/" component={GatsbyLink} {...props} />
-            )}
-            style={{
-              marginLeft: '10px',
-            }}
-          >
-            Get Listed
-          </Button>
+          {isUserLoggedIn ? (
+            <Button
+              color="primary"
+              variant="outlined"
+              size="small"
+              aria-label="Get Listed"
+              onClick={() => toggleCompanyScreen(!showCompanyScreen)}
+              style={{
+                marginLeft: '10px',
+              }}
+            >
+              Get Listed
+            </Button>
+          ) : (
+            <Button
+              color="primary"
+              variant="outlined"
+              size="small"
+              aria-label="Get Listed"
+              component={props => (
+                <Link to="/app/login/" component={GatsbyLink} {...props} />
```

Comment:
If the only difference here is whether, onClick, we're (1) showing the company create screen or (2) we're navigating the user to login, we should use the `navigate` method from `@reach/router` and just swap out that little bit of functionality rather than repeat all this code. 

It might make sense to just `handleGetListedClick` in a tiny helper function that (1) checks whether the user is authenticated and (2) acts accordingly.

Reply:
If there is a user it is a normal button. If there is no user that button actually gets replaced by that link component. So they look the same but are actually 2 different components.

Reply:
Right. But isn't the entire purpose of the Link component just to navigate the user?

Reply:
oh, gotcha ok